File: 프로그래머스/unrated/133499. 옹알이 （2）/옹알이 （2）.py
def solution(babbling):
    answer = 0
    
    words = ["aya","ye","woo","ma"]
    
    for ba in babbling:
        for w in words:
            if w*2 not in ba:
                ba = ba.replace(w,' ')
        if len(ba.strip())==0:
            print("ba =", ba)
            answer+=1
    
    return answer

File: 프로그래머스/1/133499. 옹알이 （2）/옹알이 （2）.java
import java.util.*;
class Solution {
    public int solution(String[] babbling) {
        int answer = 0;
        
        String[] vaild = {"aya", "ye", "woo", "ma"};
        
        for(String word: babbling){
            boolean isVaild = true;
            String prev = "";
            
            for(String s: vaild){
                if(word.contains(s+s)){
                    isVaild = false;
                    break;
                }
                word = word.replace(s," ");
            }
            if(isVaild&&word.trim().isEmpty()){
                answer++;
            }
        }
        
        return answer;
    }
}